flyteplugins/go/tasks/logs/config.go

```diff
@@ -28,8 +28,8 @@ type LogConfig struct {
 	StackdriverLogResourceName string              `json:"stackdriver-logresourcename" pflag:",Name of the logresource in stackdriver"`
 	StackDriverTemplateURI     tasklog.TemplateURI `json:"stackdriver-template-uri" pflag:",Template Uri to use when building stackdriver log links"`
 
-	IsFlyinEnabled   bool                `json:"flyin-enabled" pflag:",Enable Log-links to flyin logs"`
-	FlyinTemplateURI tasklog.TemplateURI `json:"flyin-template-uri" pflag:",Template Uri to use when building flyin log links"`
+	IsDynamicLogLinksEnabled bool                           `json:"dynamic-log-links-enabled" pflag:",Enable dynamic log links"`
```

Comment:
Why is this needed?

Reply:
I was just following the pattern we use for the other log types (for example k8s). 

I can see a case of someone having complex template uris that they want to temporarily disable. Do you think we should abandon that convention and rely on the configured log links instead?

Reply:
good call. I'll drop them (so they will only show up if configured).